### DISCUSSION TITLE
Issue with Features page

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#518] [Fixed Issues]
Unable to scroll to the bottom of the features page. Just flashed the newest version but cant get below "Play on Insert CD"

as always rern, THANK YOU!!!!

**Comment by feebb** [comment]
Same problem. RPi3B. Microsoft Edge on PC.

**Comment by ceejayemm** [comment]
No problem here (RPi3B+, RAudio-1 32 Bit i20210911 updated to 20210924, Win 10 Pro and MS Edge

BUT I do have' Top Bars' and 'Bars Always On' both turned on in the Playback Display options screen.

Chris

**Reply by cmh714** [reply]
I am on the latest update of 64bit, hide all the bars, and using Chrome....same thing for FF....

**Reply by ceejayemm** [reply]
See what you mean, I get the same problem if I switch to Chrome (v93.0.4577.82) but MS Edge (v93.0.961.52) seems to be OK. I don't use FF.

Chris

**Reply by cmh714** [reply]
Still happens for me with Edge too....not really a big deal for me since I dont use any of the features really, and restoring my backup sets everything appropriately. I am basically reporting so rern is aware.
